test_util/run-all

```diff
@@ -113,8 +113,16 @@ RET=$?
 mkdir -p $WORKDIR/test/vagrant-logs
 if [ $RET -ne 0 ]; then
   echo "TEST FAILED"
+  echo "Attempting to gather logs"
+  export LOG_REPLAY_LINES=${LOG_REPLAY_LINES:=1000}
```

Comment:
why does this need to be configurable? Why is it configurable only for one of the two methods?